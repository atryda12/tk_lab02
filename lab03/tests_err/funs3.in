
int a = 0, b = 0, c = 0;

int fact(int n) {
  string s = "";
  if(n==0) return s;
  else if(n==1) return s;
  else return n*fact(n-1);
}

print fact(10);

