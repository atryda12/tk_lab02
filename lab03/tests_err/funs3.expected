
Error: Improper returned type, expected int, got string: line 6
Error: Improper returned type, expected int, got string: line 7
